sitefinity-videos/sitefinity-videos-toolbar-item-provider.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable, ClassProvider, Inject } from "@angular/core";
 import { ToolBarItem, ToolBarItemsProvider, TOOLBARITEMS_TOKEN, SelectorService, SelectorOptions, SELECTOR_SERVICE } from "progress-sitefinity-adminapp-sdk/app/api/v1";
 
-require("./sf-video-toolbar.css");
+require("!style-loader!css-loader!./sitefinity-videos-toolbar-item-provider.css");
```

Comment:
Maybe "!style-loader!css-loader!" should be as a constant and used in interpolated string, so the next time we change the loaders, we don't have to change these strings.

Reply:
I prefer to leave it this way as this is the webpack specific scenario and users will find support and docs much easy directly via webpack docs (not the best, but still no need to abstract it and take support on us)

Reply:
done